recipes/msgpack-c/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "msgpack-c" %}
+{% set version = "2.1.0" %}
+{% set sha256 = "32f6ffde10c3369b67a30a9124de1e2244df188e400170c42124c62b0a06e99e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/msgpack/{{ name }}/archive/cpp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # This is needed for a bug in gcc version used where isnan is ambiguous. (::isnan and std::isnan)
+    - ambiguous_nan_inf.patch   # [linux]
+
+build:
+  number: 0
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python  # [win]
+    - vc 9    # [win and py27]
+    - vc 10   # [win and py34]
+    - vc 14   # [win and py35]
+    - gtest
+    - cmake
+    - toolchain
+    - boost 1.63.*
+    - zlib 1.2.*
+  run:
+    - vc 9   # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - boost 1.63.*
```

Comment:
Now, boost-cpp can be used. @bluescarni, can you change `boost` to `boost-cpp`? `boost-cpp` is `boost` without the python bindings.